src/tmx/CMakeLists.txt

```diff
@@ -43,8 +47,9 @@ LINK_DIRECTORIES (${PROJECT_NAME} "${TMX_LIB_DIR}")
 #
 # Add the generated SAE J2735 library and an interface that contains the build properties
 #
-ADD_SUBDIRECTORY (Asn_J2735)
-ADD_LIBRARY (libasn_j2735 ALIAS ${ASN_J2735_LIBRARIES})
+# ADD_SUBDIRECTORY (Asn_J2735)
```

Comment:
We can probably remove this line completely

Reply:
Addressed https://github.com/usdot-fhwa-OPS/V2X-Hub/pull/654/commits/ef1238953d99aa1f7aa13b59a7133c1131039ee5